src/test/java/org/jenkinsci/plugins/authorizeproject/testutil/AuthorizationCheckBuilder.java

```diff
@@ -41,7 +41,7 @@
  *
  */
 public class AuthorizationCheckBuilder extends Builder {
-    public Authentication authentication = null;
+    public transient Authentication authentication = null;

```

Comment:
Please add a comment that this `transient` is to avoid https://jenkins.io/redirect/class-filter/
as it can be confusing as it looks useless for developers who don't know JEP-200.

Reply:
You prefer such information as code comment rather than as commit comment?
I'll add it to the code too.

Reply:
Yes, I prefer that even though I know you described it in the commit comments.
Thank you.